docs/config.md

```diff
@@ -0,0 +1,16 @@
+### Config
+Set the scenarios to inject and the tunings like duration to wait between each scenario in the config file located at config/config.yaml. A sample config looks like:
+
+```
+kraken:
+    kubeconfig_path: /root/.kube/config                    # Path to kubeconfig
+    scenarios:                                             # List of policies/chaos scenarios to load
+        -    scenarios/etcd.yml
+        -    scenarios/openshift-kube-apiserver.yml
+        -    scenarios/openshift-apiserver.yml
+    node_scenarios:                                        # List of chaos node scenarios to load
+        -    scenarios/node_scenarios_example.yml
+
+tunings:
+    wait_duration: 60                                      # Duration to wait between each chaos scenario
```

Comment:
Should we add in the cerberus section and the other options in the tuning section here? 

```
cerberus:
    cerberus_enabled: False                                # Enable it when cerberus is previously installed
    cerberus_url:                                          # When cerberus_enabled is set to True, provide the url where cerberus publishes go/no-go signal

tunings:
    iterations: 1     
    daemon_mode: False  
```

Reply:
Nice catch. Fixed it, thanks.